docs/content/stable/reference/drivers/rust/rust-postgres-reference.md

```diff
@@ -0,0 +1,369 @@
+   for _i in 0..numconn {
+       let connectionresult = createconnection(url.clone()).await;
+       match connectionresult {
+           Err(error) => return Err(error),
+           Ok((connection, client)) => {
+               clientstored.push(client);
+               connectionstored.push(connection);
+           }
+       }
+   }
+   return Ok((clientstored, connectionstored));
+}
+
+async fn createconnection(url: String) -> Result<(tokio::task::JoinHandle<()>, Client), Error> {
+   let (client, connection) = yb_tokio_postgres::connect(&url, NoTls).await?;
+
+   // The connection object performs the actual communication with the database,
+   // so spawn it off to run on its own.
+   let handle = tokio::spawn(async move {
+       if let Err(e) = connection.await {
+           eprintln!("connection error: {}", e);
+       }
+   });
+
+   Ok((handle, client))
+}
+
+pub(crate) fn num_of_connections() {
+   for i in 1..4 {
+       let url = "http://127.0.0.".to_owned() + &i.to_string() + ":13000/rpcz";
+       let response = isahc::get(url);
+       if response.is_err() {
+           println!("127.0.0.{} = {}",i, 0);
+       } else {
+           let body = response.unwrap().text().unwrap();
+           let c = body.matches("client backend").count();
+           println!("127.0.0.{} = {}", i, c);
+       }
+   }
+}
+```
+
+In this case the first two nodes should have 15 connections each, and the third node should have zero connections.
+
+### Clean up
+
+After you're done experimenting, run the following command to destroy the local cluster:
+
+```sh
+./bin/yugabyted destroy --base_dir=$HOME/yugabyte-2.20.0.1/node1
+./bin/yugabyted destroy --base_dir=$HOME/yugabyte-2.20.0.1/node2
+./bin/yugabyted destroy --base_dir=$HOME/yugabyte-2.20.0.1/node3
+```
+
+## Configure SSL/TLS
+
+The YugabyteDB rust-postgres smart driver support for SSL is the same as for the upstream driver.
+
+The following table describes the connection parameters required to connect using SSL.
+
+| Parameter | Description | Default |
+| :-------- | :---------- | :------ |
+| sslmode | Controls usage of TLS. If set to:<br /> <ul><li> disable, TLS will not be used.</li><li> prefer, TLS will be used if available, but not used otherwise.</li><li> require, TLS will be forced to be used.</li></ul> | prefer |
+
+The following is an example connection URL for connecting to a YugabyteDB cluster with SSL encryption enabled:
+
+```sh
+"postgresql://127.0.0.1:5434/yugabyte?user=yugabyte&password=yugabyte&load_balance=true&sslmode=require"
+```
+
+If you created a cluster on [YugabyteDB Managed](/preview/yugabyte-cloud/), use the cluster credentials and download the [SSL Root certificate](/preview/yugabyte-cloud/cloud-connect/connect-applications/).
+
+The following is an example application for connecting to a YugabyteDB cluster with SSL enabled:
+
+Add `yb-postgres-openssl = "0.5.0-yb-1"` and `openssl = "0.10.61"` dependencies in the `Cargo.toml` file before executing the application.
```

Comment:
`yb-postgres = "0.19.7-yb-1-beta"` also needs to be added to the dependencies, had missed this before.